doc/source/ray-air/examples/torch_detection.ipynb

```diff
@@ -825,10 +825,14 @@
                 }
             ],
             "source": [
-                "from ray.air.config import DatasetConfig, ScalingConfig\n",
```

Comment:
After @raulchen removed this in https://github.com/ray-project/ray/pull/37215, this was unfortunately brought back by @bveeramani in https://github.com/ray-project/ray/pull/37178